src/openstudio_lib/IconLibrary.cpp

```diff
@@ -191,6 +191,7 @@ IconLibrary::IconLibrary() {
     new QPixmap(":images/heatpump_watertowater_equationfit_heating.png");
   m_icons[openstudio::IddObjectType(openstudio::IddObjectType::OS_Humidifier_Steam_Electric).value()] =
     new QPixmap(":images/electric_humidifier.png");
+  m_icons[openstudio::IddObjectType(openstudio::IddObjectType::OS_Humidifier_Steam_Gas).value()] = new QPixmap(":images/gas_humidifier.png");
```

Comment:
New icon looks cool, did you make it?

Reply:
I did. If you look closely, you can see pixelisation from my approximate use of Gimp. I never "create" icons. I usually take an icon, and add a portion of another icon or two or something.

electric_humidifier.png + series_fan_terminal.png

![electric_humidifier.png](https://github.com/openstudiocoalition/OpenStudioApplication/raw/develop/src/openstudio_lib/images/electric_humidifier.png)

![series_fan_terminal.png](https://github.com/openstudiocoalition/OpenStudioApplication/raw/develop/src/openstudio_lib/images/series_fan_terminal.png)